src/key_stores/browser_local_storage_key_store.ts

```diff
@@ -15,7 +15,7 @@ export class BrowserLocalStorageKeyStore extends KeyStore {
 
     /**
      * Sets a local storage item
-     * @param networkId The targeted network. (ex. default, devnet, betanet, etc…)
+     * @param networkId The targeted network. (ex. default, testnet, betanet, etc…)
```

Comment:
`default` == `testnet` right now, so I don't think this makes sense. Here and in all these other files, I think it would be better to just delete `devnet`